FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageProcessing.java

```diff
@@ -97,6 +103,14 @@ private Mat detectFeatures(Mat skeleton) {
         return results;
     }
 
+    /**
+     * Match using the ratio test and RANSAC.
+     * Returns the ratio matches and the total keypoints
+     */
+    static double matchFeatures(MatOfKeyPoint keypoints, Mat descriptors) {
+        return 1;
```

Comment:
What happens here? If this is a placeholder for a future implementation, I would just remove it.